src/basis/basis.js

```diff
@@ -396,18 +413,41 @@ export class Basis {
      */
     get maxPairwiseDistance() {
         let maxDistance = 0;
-        for (let i = 0; i < this._elements.array.length; i++) {
-            for (let j = i + 1; j < this._elements.array.length; j++) {
-                const distance = math.vDist(
-                    this._coordinates.getArrayElementByIndex(i), this._coordinates.getArrayElementByIndex(j));
-                if (distance > maxDistance) {
-                    maxDistance = distance;
+        if (this._elements.array.length >= 2) {
+            for (let i = 0; i < this._elements.array.length; i++) {
+                for (let j = i + 1; j < this._elements.array.length; j++) {
+                    const distance = math.vDist(
+                        this._coordinates.getArrayElementByIndex(i), this._coordinates.getArrayElementByIndex(j));
+                    if (distance > maxDistance) {
+                        maxDistance = distance;
+                    }
                 }
             }
         }
+        console.log(maxDistance);
         return math.precise(maxDistance, 4);
     }
 
+    /**
+     * Finds the atomic radius for an element based on it's atomic symbol (i.e. 'O' for Oxygen)
+     * The atomicRadius returned from PERIODIC_TABLE is in units of picometers (pm).
+     * To convert pm to Angstroms, multiply the pm value by 0.01.
+     *
+     * If the element does not exist inside the PERIODIC_TABLE then the atomicRadius is returned as 1.0.
+     * A value of 1.0 is chosen so that the lattice will not collapse to a size of 0, but instead will have
+     * a size of at least 2.0 angstroms once it is multipled by the lattice scaling factor for non-periodic systems.
+     *
+     * @returns {Number}
+    */
+    getElementAtomicRadius() {
+        const elementSymbol = this._elements.getArrayElementByIndex(0)
+        const elementName = Object.keys(PERIODIC_TABLE).find(key => elementSymbol === PERIODIC_TABLE[key].symbol);
+        if (elementName) {
+            return math.precise((PERIODIC_TABLE[elementName].atomic_radius_pm * 0.01), 4);
+        }
+        return 1.0;
+    }
```

Comment:
Maybe we should move the above function to the Periodic Table package, it seems like a good place for it.

Reply:
moved to periodic-table